diff -up djvulibre-3.5.27/libdjvu/DjVuPalette.cpp.omv~ djvulibre-3.5.27/libdjvu/DjVuPalette.cpp
--- djvulibre-3.5.27/libdjvu/DjVuPalette.cpp.omv~	2018-05-14 18:08:23.264642461 +0200
+++ djvulibre-3.5.27/libdjvu/DjVuPalette.cpp	2018-05-14 18:08:34.244387190 +0200
@@ -98,9 +98,11 @@ inline unsigned char
 umin(unsigned char a, unsigned char b) 
 { return (a>b) ? b : a; }
 
+#if 0 // This is in glibc...
 inline float 
 fmin(float a, float b) 
 { return (a>b) ? b : a; }
+#endif
 
 
 
